corehq/motech/dhis2/exceptions.py

```diff
@@ -0,0 +1,26 @@
+class Dhis2Exception(Exception):
+    def __init__(self, domain, base_url, username, *args, **kwargs):
```

Comment:
nit: I think `**kwargs` can be omitted since
```py
>>> Exception(message="msg")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: Exception() takes no keyword arguments
```